packages/video_player/video_player/lib/video_player.dart

```diff
@@ -563,6 +577,55 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     await _videoPlayerPlatform.setVolume(_textureId, value.volume);
   }
 
+  /// Returns true if picture-in-picture is supported on the device.
+  Future<bool> isPictureInPictureSupported() =>
+      _videoPlayerPlatform.isPictureInPictureSupported();
+
+  /// Enable/disable to start picture-in-picture automatically when the app goes to the background.
```

Comment:
Please be consistent in all of these new comments with the existing comments verb form. So this should be "Enbles/disables starting ...", the next one "Sets the location ...", etc.